service/static/js/rest_api.js

```diff
@@ -370,6 +370,35 @@ $(function () {
         clear_item_form_data()
     });
 
+    // ****************************************
+    // Retrieve an Item
+    // ****************************************
+
+    $("#retrieve-item-btn").click(function () {
+
+        let item_id = $("#order_item_id").val();
+        let order_id = $("#order_order_id").val();
+
+        var ajax = $.ajax({
+            type: "GET",
+            url: "/api/orders/" + order_id + "/items/" + item_id,
```

Comment:
You can use backticks to write it in a cleaner way as follows:

`/api/orders/${order_id}/items/${item_id}"